input.lua

```diff
@@ -26,9 +29,11 @@ return function(core, input, ...)
 	--   position 6: hello|
 
 	-- get size of text and cursor position
+
 	opt.cursor_pos = 0
 	if input.cursor > 1 then
 		local s = input.text:sub(1, utf8.offset(input.text, input.cursor)-1)
+		if input.password then s = string.rep("*", string.len(input.text)) end
```

Comment:
This line should be

`
if input.password then s = string.rep("*", utf8.len(input.text)) end
`